### PR TITLE
Login Page via Next.js 

### PR DESCRIPTION
**Pull request by fojanb**
1- A login Page with Axios POST request  "/api/login"  getting username and password
2- By inserting username and pass , the user is able to navigate to HTTP://localhost:3000/dashboard
3- Token will be stored in a localStorage (not session). Token was created by jwt.
4- For password, I have used hash via bcrypt

### CODE REVIEW

File: login-nextjs/lib/auth.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import axios from "axios";
-
+// Or: const axios = require("axios")
+// Axios ---> promise based HTTP request
 // Login the User by its username and password which returns the Response as a Promise
 export const loginUser = async (username, password) => {
 	const { data } = await axios.post("/api/login", { username, password });
```
Comment (geekforbrains) [inline login-nextjs/lib/auth.js]: I don't think you need to use Axios anymore. The built-in fetch should be more can capable. Lets update this to use fetch with async/await instead.

**Review by geekforbrains — CHANGES_REQUESTED** [review]
This is looking good! Theres a few changes and things missing related to the Jira cards:

1. Lets swap Axios for standard fetch
2. Can you switch from 4 space indents to 2? Our team standard is 2 space indents :) 
3. There is no home page where I can choose to login to register

**Comment by fojanb** [comment]
Hi Gavin , I put Login form on http://localhost:3000/auth/login



--------------------------------------------

Fojan Babaali, MASc, B.Eng

Web Developer

BC, Canada

**Comment by fojanb** [comment]
--------------------------------------------

Fojan Babaali, MASc, B.Eng

Web Developer

BC, Canada

**Pull request merged**